geode-core/src/main/java/org/apache/geode/distributed/internal/MaxLongGauge.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless rired y applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.distributed.internal;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.apache.geode.Statistics;
+
+class MaxLongGauge {
```

Comment:
Add a unit test for this new class. Some of your new tests in DistributionStats could move to this new unit test

Reply:
done.  check the tests out.  i chose to use a `StripedStatisticsImpl` as the stats instead of a mock.  It made the tests much more readable, and actually validated what we care about.  instead of checking if we called increment, it checks the actual value of the stat!!!